src/uu/free/src/free.rs

```diff
@@ -101,88 +175,198 @@ fn parse_meminfo() -> Result<MemInfo, Box<dyn std::error::Error>> {
     Ok(MemInfo::default())
 }
 
+// print total - used - free combo that is used for everything except memory for now
+// free can be negative if the memory is overcommitted so it has to be signed
+fn tuf_combo<F>(name: &str, total: u64, used: u64, free: i128, f: F)
+where
+    F: Fn(u64) -> String,
+{
+    // ugly hack to convert negative values
+    let free_str: String = if free < 0 {
+        "-".to_owned() + &f((-free) as u64)
+    } else {
+        f(free as u64)
+    };
+
+    println!("{:8}{:>12}{:>12}{:>12}", name, f(total), f(used), free_str);
+}
+
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let matches = uu_app().try_get_matches_from(args)?;
-    let wide = matches.get_flag("wide");
 
+    let wide = matches.get_flag("wide");
     let human = matches.get_flag("human");
-
     let si = matches.get_flag("si");
+    let total = matches.get_flag("total");
+    let minmax = matches.get_flag("minmax");
```

Comment:
I don't see minmax in my free binary
 free --help|grep min    
doesn't return anything ?!

Reply:
--minmax thats a new feature i implemented

Reply:
sorry but for now, we want free to be a dropped in replacement

New features should be done in a separate PR and discussed :)

Reply:
so i should remove this feature? ok :(

Reply:
yes, it needs to be discussed first
thanks